Rakefile

```diff
@@ -318,13 +330,17 @@ if false
 
 else # original
 
+  naked_mirahc_jar = new_jar.gsub(".jar","-naked.jar")
 
-  mirah_srcs = Dir['src/org/mirah/{builtins,jvm/types,macros,util,}/*.mirah'].sort +
+  mirah_srcs = Dir['src/org/mirah/{jvm/types,macros,util,}/*.mirah'].sort +
+               Dir['src/org/mirah/builtins/builtins.mirah'].sort +
```

Comment:
You could just use `['src/org/mirah/builtins/builtins.mirah']`, since we're not doing anything interesting with globs here.